WcaOnRails/spec/requests/competitions_spec.rb

```diff
@@ -57,6 +57,7 @@
     context "signed in as a delegate" do
       before :each do
         sign_in competition.delegates.first
+        competition.update!(start_date: 5.week.from_now, end_date: 5.week.from_now)
```

Comment:
Wouldn't it be better to just specify this when creating the competition at the beginning of the file?

Reply:
I guess, but I didn't want to affect any of the admin tests. Let me know if you still want this changed (I haven't included any changes for this yet).

Reply:
It shouldn't affect any of the tests, I think, but feel free to leave it as is if you think it's better! By the way, uber nit: `week`->`weeks`